ci/build_wheel_cpp.sh

```diff
@@ -25,13 +25,12 @@ rapids-pip-retry install \
 # 0 really means "add --no-build-isolation" (ref: https://github.com/pypa/pip/issues/5735)
 export PIP_NO_BUILD_ISOLATION=0
 
-export SKBUILD_CMAKE_ARGS="-DUSE_NVCOMP_RUNTIME_WHEEL=ON"
+export SKBUILD_CMAKE_ARGS="-DBUILD_SHARED_LIBS=ON;-DUSE_NVCOMP_RUNTIME_WHEEL=OFF;-DCMAKE_MESSAGE_LOG_LEVEL=VERBOSE;-DCUDA_STATIC_RUNTIME=ON"
```

Comment:
Do we need to change all of these? Or just `-DUSE_NVCOMP_RUNTIME_WHEEL`? The verbose logging one seems safe to remove.
```suggestion
export SKBUILD_CMAKE_ARGS="-DBUILD_SHARED_LIBS=ON;-DUSE_NVCOMP_RUNTIME_WHEEL=OFF;-DCUDA_STATIC_RUNTIME=ON"
```